uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Page/AbsenceTab.php

```diff
@@ -6,6 +6,22 @@
 class CRM_HRLeaveAndAbsences_Page_AbsenceTab extends  CRM_Core_Page {
   public function run() {
     CRM_Utils_System::setTitle(ts('Absence'));
+
+    self::registerScripts();
     parent::run();
   }
+
+  static function registerScripts() {
+    static $loaded = FALSE;
+
+    if ($loaded) {
+      return;
+    }
+
+    $loaded = TRUE;
+
+    //TODO find a better way to get path to theme
+    CRM_Core_Resources::singleton()
+      ->addStyleUrl('http://'.$_SERVER['HTTP_HOST'].'/sites/all/themes/civihr_employee_portal_theme/civihr_default_theme/assets/css/civihr_default_theme.style.css');
```

Comment:
There are a few problems here:
- the protocol is hardcoded to http. This will cause problems if the site is served using https. To avoid this, remove the `http:`
- You're accessing a super global directly, which is usually a bad practice and can cause problems if the host was changed somehow.

To address these problems, I suggest you to add the css file like this:
```php
$sspCssFilePath = 'sites/all/themes/civihr_employee_portal_theme/civihr_default_theme/assets/css/civihr_default_theme.style.css';
$sspCssFileUrl = CRM_Utils_System::url($sspCssFilePath);
CRM_Core_Resources::singleton()->addStyleUrl($sspCssFile);
```